### PR TITLE
Update Homepage

### PR DESCRIPTION
**Pull request by mnicolau9**
Changes up on https://developer-stage.adobe.com/adobe-pass/

### CODE REVIEW

File: src/pages/index.md [hunk 1]
```diff
@@ -5,125 +5,89 @@ description: This is the overview page of Cat Analytics
 #  - https://github.com/icaraps 
 ---
 
-<Hero slots="heading, text"></Hero>
+<Hero slots="heading, text" background="rgb(246, 16, 27)"></Hero>
```
Comment (ancad-adobe) [inline src/pages/index.md:8]: remove red colour

File: src/pages/index.md [hunk 2]
```diff
@@ -5,125 +5,89 @@ description: This is the overview page of Cat Analytics
...
-The scenario in which the primary streaming application has the capability to create an authentication session, but does not have the ability to gather all the required parameters or open a user agent in order to authenticate with the selected MVPD.
-
-##### [Authorization](https://experienceleague.adobe.com/en/docs/pass/authentication/programmer-integration-guide/rest-api-v2/rest-api-v2-flows/rest-api-v2-basic-flows/rest-api-v2-basic-authorization-primary-application-flow.html)
-
-Verify that a user is entitled to view content represented as resource from the primary streaming application.
-
-##### [Preauthorization](https://experienceleague.adobe.com/en/docs/pass/authentication/programmer-integration-guide/rest-api-v2/rest-api-v2-flows/rest-api-v2-basic-flows/rest-api-v2-basic-preauthorization-primary-application-flow.html)
-
-Verify that a user may have access to a list of resources in order for the primary streaming application to present accurate information to the user on content he would be able to see.
+* [Adobe Pass Authentication Documentation](https://experienceleague.adobe.com/en/docs/pass/authentication/home)
```
Comment (ancad-adobe) [inline src/pages/index.md:31]: remove 'Adobe' [Pass Authentication Documentation](https://experienceleague.adobe.com/en/docs/pass/authentication/home)
Comment

File: src/pages/index.md [hunk 3]
```diff
@@ -5,125 +5,89 @@ description: This is the overview page of Cat Analytics
...
+* [REST API v2 APIs Overview](https://experienceleague.adobe.com/en/docs/pass/authentication/programmer-integration-guide/rest-api-v2/rest-api-v2-apis/rest-api-v2-apis-overview)
```
Comment (ancad-adobe) [inline src/pages/index.md:32]: v2 -> V2

File: src/pages/index.md [hunk 4]
```diff
@@ -5,125 +5,89 @@ description: This is the overview page of Cat Analytics
...
+* [REST API v2 Flows Overview](https://experienceleague.adobe.com/en/docs/pass/authentication/programmer-integration-guide/rest-api-v2/rest-api-v2-flows/rest-api-v2-flows-overview)
```
Comment (ancad-adobe) [inline src/pages/index.md:33]: v2 -> V2

File: src/pages/index.md [hunk 5]
```diff
@@ -5,125 +5,89 @@ description: This is the overview page of Cat Analytics
...
 <AnnouncementBlock slots="heading, text, button"></AnnouncementBlock>
 
-### Use the interactive API environment
+### Adobe Pass Authentication
 
-You can now interact with the Adobe Pass Authentication API endpoints directly from the API reference pages. Get an authorization through our [DCR API](api/dcr_api) and click the button below to interact with our REST API.
+Integrate the new REST API v2 to benefit from the latest Adobe Pass functionalities.
 
-[Try our REST API](api/rest_api_v2/interactive)
+[Try REST API v2](api/rest_api_v2/interactive)
```
Comment (ancad-adobe) [inline src/pages/index.md:25]: v2 -> V2

File: src/pages/index.md [hunk 6]
```diff
@@ -5,125 +5,89 @@ description: This is the overview page of Cat Analytics
...
+#### Unique implementation across all platforms
 
-Perform authentication through a Partner that is providing its own partner framework for implementing the authentication flow with supported MVPDs.
+Customers' applications can now use the same implementation across platforms, making it easier to launch new features or maintain live apps.
 
-##### [Platform Identity SSO Authentication](https://experienceleague.adobe.com/en/docs/pass/authentication/programmer-integration-guide/rest-api-v2/rest-api-v2-flows/rest-api-v2-single-sign-on-flows/rest-api-v2-single-sign-on-platform-identity-flows.html)
+#### Cross-device SSO
 
-This single sign-on method is based on sharing a unique identifier which could be a device scoped identifier or platform scoped identifier. This unique identifier can be used from multiple applications to perform single sign-on flows at device / platform level.
+REST API v2 allows an authentication session to be securely passed between different devices. By simply passing the session between devices, users can authenticate on their mobile device and stream video on a TV-connected device without re-authentication.
```
Comment (ancad-adobe) [inline src/pages/index.md:59]: v2 -> V2

File: src/pages/index.md (hunk 3 above)
Comment (ancad-adobe) [inline src/pages/index.md:32]: This page should refer Developer Website from Experience League. No need to refer here

**Review by ancad-adobe — CHANGES_REQUESTED** [review]

File: src/pages/index.md (hunk 3 above)
Comment (mnicolau9) [reply src/pages/index.md:32]: I think we need to keep a direct reference to Experience League's REST V2 documentation, as the first page is a general landing page for all our APIs. Also, it is my understanding that the API Overview page is subject to change and may no longer refer Developer Website in a future update

File: src/pages/index.md (hunk 2 above)
Comment (ancad-adobe) [reply src/pages/index.md:31]: If we keep the actual length, it's not going to be rendered right

File: src/pages/index.md (hunk 1 above)
Comment (ancad-adobe) [reply src/pages/index.md:8]: or change it like here: https://developer.adobe.com/firefly-services/docs/guides/

**Review by ancad-adobe — APPROVED** [review]

**Pull request merged**